libmscore/measure.cpp

```diff
@@ -1734,7 +1734,7 @@ void Measure::read(XmlReader& e, int staffIdx)
                   breath->setTrack(e.track());
                   int tick = e.tick();
                   breath->read(e);
-                  if (score()->mscVersion() < 205) {
+                  if (score()->mscVersion() <= 206) {
                         // older scores placed the breath segment right after the chord to which it applies
```

Comment:
Outch! I though this would not be any functional Change (I apparently read the `<` as a `>`, now I see that it is.